tools/dependency/release_dates.py

```diff
@@ -111,6 +134,51 @@ def issues_exist(title, git):
     return issues.totalCount > 0
 
 
+# search for issue by title and delete old issue if new package version is available
+def search_old_version_open_issue_exist(title, git, package_repo, latest_release):
+    # search for only "Newer release available {dep}:" as will be common in dep issue
+    title_search = title[0:title.index(":")]
+    query = f'repo:{GITHUB_REPO_LOCATION} {title_search} in:title is:open'
+    # there might be more than one issue
+    # if current package version == issue package version no need to do anything, right issue is open
+    # if current package version != issue_title_version means a newer updated version is available
+    # and close old issue
+    issues = git.search_issues(query)
+    for issue in issues:
+        issue_version = get_package_version_from_issue(issue.title)
+        if issue_version != latest_release.tag_name:
+            close_old_issue(git, issue.number, latest_release, package_repo)
+
+
+def get_package_version_from_issue(issue_title):
+    # issue title create by github action has two form
+    if "(" in issue_title:
```

Comment:
i think this could be simpler - and not require the condition etc - perhaps:

```
issue_title.split(":")[1].split("(")[0].strip()
```

Reply:
Hmm smooth. Ok will replace it